base_duplicate_security_group/static/src/js/duplicate.esm.js

```diff
@@ -11,11 +11,14 @@ import session from "web.session";
  * If the user has the permission, the internal logic rules will apply.
  **/
 export const DuplicateViewMixin = {
-    async willStart() {
-        this.controllerParams.activeActions.duplicate = await session.user_has_group(
-            "base_duplicate_security_group.group_duplicate_records"
-        );
-        return this._super(...arguments);
+    init() {
+        this._super(...arguments);
+        const base_group = "base_duplicate_security_group.group_duplicate_records";
+        session.user_has_group(base_group).then((result) => {
+            if (!result) {
+                this.controllerParams.activeActions.duplicate = false;
+            }
```

Comment:
Is `this.controllerParams.activeActions.duplicate` activated without defining it as true?

Reply:
Yup -> https://github.com/odoo/odoo/blob/1785047d58f28ba11e11e40ba134e89bb235fa8b/addons/web/static/src/legacy/js/views/abstract_view.js#L146